.github/workflows/release.yml

```diff
@@ -0,0 +1,91 @@
+name: CI Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ windows-latest ] 
+        dotnet: ['5.0.x']
+
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v2
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: ${{ matrix.dotnet }}
+
+      - name: Install dependencies
+        run: dotnet restore
+      - name: Build
+        run: dotnet build --configuration Release --no-restore
+      - name: Unit test
+        run: dotnet test Amazon.IonObjectMapper.Test
```

Comment:
Wouldnt this already be done by the standard actions?

Reply:
Yes, but I feel like it wouldn't hurt to add this here.

I am also following the Rust shell's [release GH actions](https://github.com/awslabs/amazon-qldb-shell/blob/rust/.github/workflows/release.yml#L13)